great_expectations/data_context/types/resource_identifiers.py

```diff
@@ -184,9 +184,6 @@ def resource_identifier(self):
         return self._resource_identifier
 
     def to_tuple(self):
-        # if PY3:
-        #     return (self.site_section_name, *self.resource_identifier.to_tuple())
-        # else:
```

Comment:
Not live code but seems fine to delete.  I think we could use the syntax in the `if PY3:` block here if we want to but it looks like the current pattern is being used in a similar manner elsewhere, so left it as-is for consistency.